### ISSUE TITLE
Create demo environment - Terraform

### ISSUE THREAD

**Issue by jeremy-page** [CDCgov/prime-devops#58]
As a ReportStream engineer I would like a demo environment that I deploy an arbitrary branch to for end user testing. 

- This environment does not need to connect to the CDC network
- For now we can assume users will be authenticated via Okta
- Should mimic the test environment as closely as possible (but with the string `demo`)
- This ticket covers the Terraform changes

**Comment by JosiahSiegel** [comment]
The Reportstream “demo environment pipeline” is nearly complete!
While I will be unavailable next week :vacation:, I am leaving 1 Azure demo environment running in resource group [prime-data-hub-demo2](https://portal.azure.com/#@cdc.onmicrosoft.com/resource/subscriptions/7d1e3999-6577-4cd5-b296-f518e5c8e677/resourceGroups/prime-data-hub-demo2/overview) for those interested in taking a look and/or making recommendations.
We will still need to solve for creating certificates and Okta integration (https://pdhdemo2public.z13.web.core.windows.net/)

Overview of demo pipeline attached.
In short, run a pipeline (or command) to provision 1 of 3 demo environments, and run a pipeline (or command) to destroy it.
![Demo Environments](https://user-images.githubusercontent.com/5522990/176967239-4b73b52f-8865-48a6-b470-d0b155f7b859.png)

**Comment by JosiahSiegel** [comment]
Submitted email to update front door certs to include demo envs:



<body link="#0563C1" vlink="#954F72">



Open Ticket   Notification |  
-- | --
Incident IM6108444 has been Opened. |  
DETAILS |  
Incident Number | IM6108444
Contact Name | Siegel, Josiah (CDC/DDPHSS/OD/HITSSU)
User ID | TEY1
Location | CENT Bldg 2500 Rm 5200/MS V25-5




</body>

</html>
